### ISSUE TITLE
Prefix hydra arguments (one more minor improvement)

### ISSUE THREAD

**Issue by vkazanov** [abo-abo/hydra#21]
One of the issues fixed a few days ago included prefix argument use *within* a hydra. This is great and certainly helps a lot. I switched to hydra dozens of my custom commands.

But there is a problem with my finger memory: usual non-hydra Emacs commands accept prefix args *before* the prefix. Think `C-u C-z s`, where `C-z s` is bound to a command, which accepts a universal argument. 

Now, when the command is within the hydra, I have to the following: `C-z C-u s. This conflicts with my muscle memory, and I believe this is typical for most Emacs users.

Maybe it is possible to pass the hydra args to the first command? So I can do something like `C-u C-z s C-u s`?

**Comment by abo-abo** [comment]
It get what you're saying; the implementation doesn't seem straightforward, I'll have to think
about this for a while.

Could you provide a minimal hydra and a minimal scenario that you wish would work?

**Comment by vkazanov** [comment]
Sure. Here goes a simplified hydra from my .emacs:

```
(global-set-key
 (kbd "C-z")
 (defhydra vlk ()
   "Vlk"
   ("q" nil "cancel")

   ("j"  org-journal-new-entry "new entry")))
```
I removed non-relevant parts of the hydra (related to navigation). 

`org-journal-new-entry` normally creates a new journal entry. With a prefix arg it switches to the latest entry without creating a new one. I used to bind it to `C-z j`, so to get to the latest entry I usually just do `C-u C-z j`.

With the hydra in place I do `C-z j` to create an entry and `C-z C-u j` to get back to it, which definitely feels similar, but breaks the way emacsers deal with prefix args.
